packages/components/form/src/form-item.vue

```diff
@@ -303,7 +303,7 @@ const validate: FormItemContext['validate'] = async (trigger, callback) => {
   }
 
   const rules = getFilteredRule(trigger)
-  if (rules.length === 0) {
+  if (rules.length === 0 || rules.every((rule) => !rule.required)) {
```

Comment:
**Nitpick:** perhaps we could do:

```suggestion
  if (rules.length === 0 || !rules.some((rule) => rule.required)) {
```

If any rule contains `required == true`, then this expression should be false thus the condition won't hold. WDTY?

Reply:
According to the second point mentioned below, if required is set to false, it should also be validate. So the judgment here may not be necessary, we should close this pr.